02/MirrorString.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include<string>
+#include "MirrorString.h"
+
+using namespace std;
+
+namespace samples
+{
+	void MirrorStringExample()
+	{
+		string line = "Hello World!";
+
+		for (size_t i = line.length(); i--> 0;)
```

Comment:
oh yeah probably u have to do int i... cuz size_t is  unisnged so when it goes from 0 to -1 it actually becomes size_t::max...

sorry